src/frontend/src/common/utils/featureFlags.tsx

```diff
@@ -0,0 +1,24 @@
+import { hasQueryParam } from ".";
+
+interface FeatureFlag {
+  isActive: boolean;
```

Comment:
This is a quibble, but I think `isActive` leads to the assumption that the flag is globally on, which is not the case. It's more like "enabled, assuming the user also has passed the matching query param". I think it might be less prone to misinterpretation to go the opposite direction and instead call it `disabled`, then flip the `if isActive` to `if !disabled`.